ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/sorter/AbstractPendingTransactionsSorter.java

```diff
@@ -138,6 +141,39 @@ public void evictOldTransactions() {
             });
   }
 
+  /**
+   * Evaluates the transaction pool and evicts future nonce transactions until the global count of
+   * future nonce transactions is below the configured limit.
+   *
+   * <p>Eviction criteria is first distance from the current account nonce, and then age in the
```

Comment:
Does this effectively evict largest distance first, then oldest first? Might be worth clarifying that here.